docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
-([boot2docker](https://github.com/boot2docker/boot2docker)) with the Docker
-daemon installed, and will create and start a VirtualBox VM with Docker running.
-
-
-```
-$ docker-machine create --driver virtualbox dev
-Creating CA: /home/username/.docker/machine/certs/ca.pem
-Creating client certificate: /home/username/.docker/machine/certs/cert.pem
-Image cache does not exist, creating it at /home/username/.docker/machine/cache...
-No default boot2docker iso found locally, downloading the latest release...
-Downloading https://github.com/boot2docker/boot2docker/releases/download/v1.6.2/boot2docker.iso to /home/username/.docker/machine/cache/boot2docker.iso...
-Creating VirtualBox VM...
-Creating SSH key...
-Starting VirtualBox VM...
-Starting VM...
-To see how to connect Docker to this machine, run: docker-machine env dev
-```
-
-You can see the machine you have created by running the `docker-machine ls`
-command again:
-
-```
-$ docker-machine ls
-NAME   ACTIVE   DRIVER       STATE     URL                         SWARM
-dev             virtualbox   Running   tcp://192.168.99.100:2376
-```
+Docker Machine supports over fourteen providers including Amazon Web Services,
+Azure, Digital Ocean, and more. It also supports a generic provider if you want
+to add your own provider.
 
-Next, as noted in the output of the `docker-machine create` command, we have to
-tell Docker to talk to that machine. You can do this with the `docker-machine
-env` command. For example,
+After Machine has your Docker Host up and running, the second step is to
+direct your command-line shell to work with your new machine:
 
+```bash
+$ eval "$(docker-machine env my-docker-machine)"
 ```
-$ eval "$(docker-machine env dev)"
-$ docker ps
-```
-
-> **Note**: If you are using `fish`, or a Windows shell such as
-> Powershell/`cmd.exe` the above method will not work as described. Instead,
-> see [the `env` command's documentation](https://docs.docker.com/machine/#env)
-> to learn how to set the environment variables for your shell.
 
-This will set environment variables that the Docker client will read which specify
-the TLS settings. Note that you will need to do that every time you open a new tab or
-restart your machine.
+This command sets a number of variables:
 
-To see what will be set, run `docker-machine env dev`.
-
-```
-$ docker-machine env dev
+```bash
 export DOCKER_TLS_VERIFY="1"
-export DOCKER_HOST="tcp://172.16.62.130:2376"
-export DOCKER_CERT_PATH="/Users/<your username>/.docker/machine/machines/dev"
-export DOCKER_MACHINE_NAME="dev"
-# Run this command to configure your shell:
-# eval "$(docker-machine env dev)"
+export DOCKER_HOST="tcp://192.168.99.100:2376"
+export DOCKER_CERT_PATH="/Users/mary/.docker/machine/machines/my-docker-machine"
+export DOCKER_MACHINE_NAME="my-docker-machine"
 ```
 
-You can now run Docker commands on this host:
+The `DOCKER_TLS_VERIFY` sets transport layer security (TLS) on communication
```

Comment:
Actually, in my experience with other Docker products, they are very interested. Since Docker has a reputation for being insecure, it also makes sense from a Product Marketing perspective to point this default behavior out.